.env_SAMPLE

```diff
@@ -72,7 +73,7 @@ export SHEER_ELASTICSEARCH_INDEX=content
 
 # Name of the virtualenv created for Sheer in
 # https://github.com/cfpb/cfgov-refresh/blob/flapjack/INSTALL.md#sheer
-export VIRTUAL_ENV=cfgov-refresh
+export VENV_NAME=testing
```

Comment:
What's wrong with `cfgov-refresh`? `testing` seems too generic to me. I have a lot of virtualenvs.